docs/api/openapi3.json

```diff
@@ -2203,14 +2187,14 @@
         "operationId": "GetIdentity",
         "parameters": [
           {
-            "example": "4yJ3n3D8E2",
             "in": "path",
             "name": "id",
             "required": true,
             "schema": {
+              "description": "a uid or the literal self",
               "example": "4yJ3n3D8E2",
               "format": "uid",
```

Comment:
I think this format specifier should be `"uid|self" too

Reply:
I'm not exactly sure how to represent a description of a format vs a literal, I'll use `uid|self` for now and we can figure it out later.